lib/fetcher.py

```diff
@@ -1,11 +1,18 @@
 # Fetches tickers of stocks listed on Indian Stock Exchanges
-# returns the first `limit` tickers
+# returns the first `limit` ticker data
 
-def fetch(session: [requests.sessions.Session], limit: int) -> [pd.DataFrame]:
+import requests
+import pandas as pd
+import re
+from time import sleep
+from bs4 import BeautifulSoup
+from typing import List
+
+def fetch(session: requests.Session, limit: int) -> List[pd.DataFrame]:
```

Comment:
What is the purpose of setting the function return type to List[pd.DataFrame]? The output type is a pd.DataFrame with 3 columns for now.

Reply:
Done! @AshutoshSancheti